sched/sched/sched_note.c

```diff
@@ -494,7 +494,7 @@ void sched_note_start(FAR struct tcb_s *tcb)
   namelen = strlen(tcb->name);
 
   DEBUGASSERT(namelen <= CONFIG_TASK_NAME_SIZE);
-  strncpy(note.nsa_name, tcb->name, CONFIG_TASK_NAME_SIZE + 1);
+  strlcpy(note.nsa_name, tcb->name, CONFIG_TASK_NAME_SIZE + 1);
```

Comment:
Should we remove +1 now?

Reply:
no, since note.nsa_name contain `CONFIG_TASK_NAME_SIZE + 1`'s char,

Reply:
Then maybe switch to sizeof?